src/gsy_e/models/strategy/external_strategies/__init__.py

```diff
@@ -367,11 +367,13 @@ def _offer_aggregator_impl(
                 "offer": offer.to_json_string(replace_existing=replace_existing),
                 "transaction_id": arguments.get("transaction_id"),
                 "area_uuid": self.device.uuid,
+                "market_type": market.__class__.__name__,
```

Comment:
I would recommend creating a property for this to hide the implementation detail and the nested calls. I would suggest for example `type_name` (since `type` is a reserved keyword in Python) or `type_`